src/docs/containers/color-palette.jsx

```diff
@@ -211,13 +211,55 @@ const colorList = [
 	},
 ];
 
+// Yoinked from SO: http://stackoverflow.com/a/3627747/895558
+function hex(x) {
+	return ('0' + parseInt(x).toString(16)).slice(-2);
+}
+
+function rgb2hex(rgb) {
+	const matches = rgb.match(/^rgb\((\d+),\s*(\d+),\s*(\d+)\)$/);
+
+	return '#' + hex(matches[1]) + hex(matches[2]) + hex(matches[3]);
+}
+
 const ColorPalette = React.createClass({
+
+	// In order to keep our colors coming from LESS, we need to do some...
+	// interesting... dom inspection to get the actual background colors
+	// rendered.
+	injectHexDisplays() {
+		const allItems = document.querySelectorAll(`.${cx('&-item')}`);
+
+		_.each(allItems, (item) => {
```

Comment:
probably not worth it now, but couldn't you change this to a `map` and throw the list into state, so render could just access them?